### PR TITLE
Shorten the overrides example

### PR DESCRIPTION
**Pull request by johnwalker**
This adds an example that adds an encrypted record using the mapper, and loads from backup using encryption context operators.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by juneb** [comment]
When this PR is merged, I'll link to it from the DDB Encryption Client docs.

File: examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java [hunk 1]
```diff
@@ -48,107 +48,73 @@ public static void main(String[] args) throws GeneralSecurityException {
...
-        );
+        encryptor.setEncryptionContextOverrideOperator(EncryptionContextOperators.overrideEncryptionContextTableName(
+                dynamoBackupName, dynamoTableName));
 
-        // Mapper Creation
-        // Please note the use of SaveBehavior.PUT (SaveBehavior.CLOBBER works as well).
-        // Omitting this can result in data-corruption.
-        DynamoDBMapperConfig mapperConfig = DynamoDBMapperConfig.builder()
-                .withSaveBehavior(DynamoDBMapperConfig.SaveBehavior.PUT).build();
         DynamoDBMapper mapper = new DynamoDBMapper(ddb, mapperConfig, new AttributeEncryptor(encryptor));
```
Comment (mattsb42-aws) [inline examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java]: You deleted the creation of `mapperConfig` but did not replace it with anything.

File: examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java [hunk 2]
```diff
@@ -48,107 +48,73 @@ public static void main(String[] args) throws GeneralSecurityException {
...
+
+        DuckPojo duckPojo = new DuckPojo();
+        duckPojo.setDuckName("Quackley");
+        duckPojo.setSwimming(true);
 
-        // Set up our configuration and clients
         final DirectKmsMaterialProvider cmp = new DirectKmsMaterialProvider(kms, cmkArn);
         final DynamoDBEncryptor encryptor = DynamoDBEncryptor.getInstance(cmp);
 
-        Map<String, String> tableNameEncryptionContextOverrides = new HashMap<>();
```
Comment (mattsb42-aws) [inline examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java]: If this is the only example we have for table name overrides, can we include this as, say, a commented describing or at least referencing the `overrideEncryptionContextTableNameUsingMap` option?

File: examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java (hunk 1 above)
Comment (johnwalker) [reply examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java]: It is replaced with a static instance. Yes, it is tested.

File: examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java (hunk 2 above)
Comment (johnwalker) [reply examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java]: Yes, that sounds like a good idea.

File: examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java (hunk 1 above)
Comment (mattsb42-aws) [reply examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java]: Whoops; that missed my text search.

File: examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java (hunk 2 above)
Comment (johnwalker) [reply examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java]: Added a comment to mention `overrideEncryptionContextTableNameUsingMap`

**Comment by ragona** [comment]
Resolving the outstanding requested changing changes; looks like they're complete.

**Comment by johnwalker** [comment]
Yes, this code is tested. (I commented on that question, but somehow I can't see it anymore)

**Comment by johnwalker** [comment]
Closing, stale.

**Pull request closed without merging**